modules/data.atmosphere/R/GEFS_helper_functions.R

```diff
@@ -0,0 +1,906 @@
+#' Download gridded forecast in the box bounded by the latitude and longitude list
+#'
+#' @param lat_list
+#' @param lon_list
+#' @param forecast_time
+#' @param forecast_date
+#' @param model_name_raw
+#' @param num_cores
+#' @param output_directory
+#'
+#' @return
+#' @export
+#'
+#' @examples
+noaa_grid_download <- function(lat_list, lon_list, forecast_time, forecast_date ,model_name_raw, output_directory, end_hr) {
+  
+  
+  download_neon_grid <- function(ens_index, location, directory, hours_char, cycle, base_filename1, vars,working_directory){
```

Comment:
There's a lot of references to NEON in this script, including in file naming conventions, which doesn't make sense to me for a general NOAA download function. I know a lot of this is borrowed from the NEON challenge, but this will be confusing in the future. Plus we want to have filenaming keep to established PEcAn patterns.

Reply:
I removed the neon references